packages/gatsby/cache-dir/__tests__/dev-loader.js

```diff
@@ -372,7 +372,7 @@ describe(`Dev loader`, () => {
 
     it(`should return an error when component cannot be loaded`, async () => {
       const syncRequires = createSyncRequires({
-        chunk: false,
+        chunk: Promise.resolve(false),
```

Comment:
this seems sketchy - I would get failing test if this was just `false`, but then with `syncRequires` it doesn't make much sense to use test as above as it's not a promise ...